### ISSUE TITLE
[laravel-queue] Package compatibility issue with Laravel 7

### ISSUE THREAD

**Issue by nospoon** [php-enqueue/enqueue-dev#1073]
When trying to install the package with Laravel 7, composer throws an error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for enqueue/laravel-queue ^0.10.2 -> satisfiable by enqueue/laravel-queue[0.10.2].
    - Conclusion: remove ramsey/uuid 4.0.1
    - Conclusion: don't install ramsey/uuid 4.0.1
    - enqueue/laravel-queue 0.10.2 requires enqueue/enqueue ^0.10 -> satisfiable by enqueue/enqueue[0.10.0, 0.10.1, 0.10.x-dev].
    - enqueue/enqueue 0.10.0 requires ramsey/uuid ^2|^3.5 -> satisfiable by ramsey/uuid[2.0.0, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.3.0, 2.4.0, 2.5.0, 2.6.0, 2.6.1, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 3.5.0, 3.5.1, 3.5.2, 3.6.0, 3.6.1, 3.7.0, 3.7.1, 3.7.2, 3.7.3, 3.8.0, 3.9.0, 3.9.1, 3.9.2, 3.9.3, 3.x-dev].
    - enqueue/enqueue 0.10.1 requires ramsey/uuid ^2|^3.5 -> satisfiable by ramsey/uuid[2.0.0, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.3.0, 2.4.0, 2.5.0, 2.6.0, 2.6.1, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 3.5.0, 3.5.1, 3.5.2, 3.6.0, 3.6.1, 3.7.0, 3.7.1, 3.7.2, 3.7.3, 3.8.0, 3.9.0, 3.9.1, 3.9.2, 3.9.3, 3.x-dev].
    - enqueue/enqueue 0.10.x-dev requires ramsey/uuid ^2|^3.5 -> satisfiable by ramsey/uuid[2.0.0, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.3.0, 2.4.0, 2.5.0, 2.6.0, 2.6.1, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0, 3.5.0, 3.5.1, 3.5.2, 3.6.0, 3.6.1, 3.7.0, 3.7.1, 3.7.2, 3.7.3, 3.8.0, 3.9.0, 3.9.1, 3.9.2, 3.9.3, 3.x-dev].
    - Can only install one of: ramsey/uuid[3.7.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.7.1, 4.0.1].
    - Can only install one of: ramsey/uuid[3.7.2, 4.0.1].
    - Can only install one of: ramsey/uuid[3.7.3, 4.0.1].
    - Can only install one of: ramsey/uuid[3.8.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.9.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.9.1, 4.0.1].
    - Can only install one of: ramsey/uuid[3.9.2, 4.0.1].
    - Can only install one of: ramsey/uuid[3.9.3, 4.0.1].
    - Can only install one of: ramsey/uuid[3.x-dev, 4.0.1].
    - Can only install one of: ramsey/uuid[2.0.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.1.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.1.1, 4.0.1].
    - Can only install one of: ramsey/uuid[2.1.2, 4.0.1].
    - Can only install one of: ramsey/uuid[2.2.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.3.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.4.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.5.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.6.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.6.1, 4.0.1].
    - Can only install one of: ramsey/uuid[2.7.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.7.1, 4.0.1].
    - Can only install one of: ramsey/uuid[2.7.2, 4.0.1].
    - Can only install one of: ramsey/uuid[2.7.3, 4.0.1].
    - Can only install one of: ramsey/uuid[2.7.4, 4.0.1].
    - Can only install one of: ramsey/uuid[2.8.0, 4.0.1].
    - Can only install one of: ramsey/uuid[2.8.1, 4.0.1].
    - Can only install one of: ramsey/uuid[2.8.2, 4.0.1].
    - Can only install one of: ramsey/uuid[2.8.3, 4.0.1].
    - Can only install one of: ramsey/uuid[2.8.4, 4.0.1].
    - Can only install one of: ramsey/uuid[2.9.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.5.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.5.1, 4.0.1].
    - Can only install one of: ramsey/uuid[3.5.2, 4.0.1].
    - Can only install one of: ramsey/uuid[3.6.0, 4.0.1].
    - Can only install one of: ramsey/uuid[3.6.1, 4.0.1].
    - Installation request for ramsey/uuid (locked at 4.0.1) -> satisfiable by ramsey/uuid[4.0.1].

```

**Comment by Gounlaf** [comment]
It's related to #1036 #1057

**Comment by Dragomeat** [comment]
Why is there still no support for uuid v4 in enqueue? Is this project dead?

**Comment by Dragomeat** [comment]
@makasim Can you please answer this question?

**Comment by makasim** [comment]
I've just released 0.10.3. Let me know if this fixes the issue.

**Comment by makasim** [comment]
If not, please submit a fix.
